types/index.d.ts

```diff
@@ -78,6 +78,10 @@ declare namespace fastifySwaggerUi {
   type FastifySwaggerUILogo = {
     type: string;
     content: string | Buffer;
+    href?: {
+      url: string;
+      target_blank?: boolean;
+    }
```

Comment:
I would actually say, that the logo gets a href and a target and not an object

```suggestion
    href?: string;
    target?: '_blank' | '_parent' | '_self' | '_top';
```